apps/explorer/src/app/components/txs/txs-per-block.tsx

```diff
@@ -17,15 +17,11 @@ interface TxsPerBlockProps {
 const truncateLength = 5;
 
 export const TxsPerBlock = ({ blockHeight, txCount }: TxsPerBlockProps) => {
-  // TODO after https://github.com/vegaprotocol/vega/pull/6958/files is merged and deployed, use filter
-  // by block height instead
+  const filters = `filters[block.height]=${blockHeight}`;
+  const url = getTxsDataUrl({ limit: txCount.toString(), filters });
```

Comment:
suggestion (non blocking): hooks should be named with the useX convention

Reply:
Oh this is just a function within the file. Super minor but perhaps move to a more general helpers file?

Reply:
I'll consider that next time I call it. So far it's only used twice.